lib/src/configuration.dart

```diff
@@ -97,6 +102,11 @@ class Configuration {
   /// The realm file must already exists at [path]
   bool get isReadOnly => realmCore.getConfigReadOnly(this);
   set isReadOnly(bool value) => realmCore.setConfigReadOnly(this, value);
+
+  /// Gets or sets a value specifying settings for an in-memory Realm.
+  /// When all in-memory instance of Realm is closed all data in that Realm is deleted.
+  bool get inMemory => realmCore.getConfigInMemory(this);
+  set inMemory(bool value) => realmCore.setConfigInMemory(this, value);
```

Comment:
This name diverges with `isReadOnly` we could syncing it. So `isInMemory` or we rename `isReadOnly` to `readOnly`

Reply:
Renamed to isInMemory .